lib/matchers/hashmap.js

```diff
@@ -26,7 +26,7 @@ module.exports = factory({
 
     var errors = [];
     if (this.matchers.keys) {
-      var keyErrors = new s.array({of: this.matchers.keys}).match(path + '.keys', Object.keys(obj));
+      var keyErrors = s.array({of: this.matchers.keys}).match(path + '.keys', Object.keys(obj));
```

Comment:
@nguyenchr 
I found something which looks like the hashmap, not 100% match, so I am not quite sure if this fits for the hashmap.

```js
{
    "$schema": "http://json-schema.org/draft-04/schema#",
    "type": "object",
    "properties": {
        "/": { "$ref": "http://some.site.somewhere/entry-schema#" }
    },
    "patternProperties": {
        "^(/[^/]+)+$": { "$ref": "http://some.site.somewhere/entry-schema#" }
    },
    "additionalProperties": false,
    "required": [ "/" ]
}
```